flink-runtime/src/test/java/org/apache/flink/runtime/scheduler/SsgNetworkMemoryCalculationUtilsTest.java

```diff
@@ -70,6 +70,26 @@ public class SsgNetworkMemoryCalculationUtilsTest {
 
     @Test
     public void testGenerateEnrichedResourceProfile() throws Exception {
+        testGenerateEnrichedResourceProfile(
+                ResultPartitionType.PIPELINED,
+                new MemorySize(
+                        TestShuffleMaster.computeRequiredShuffleMemoryBytes(0, 2)
+                                + TestShuffleMaster.computeRequiredShuffleMemoryBytes(1, 12)),
+                new MemorySize(TestShuffleMaster.computeRequiredShuffleMemoryBytes(10, 0)));
+
+        testGenerateEnrichedResourceProfile(
```

Comment:
It's better to add some comments to illustrate why this value is expected. This is because after all these changes it is not easy for someone unfamiliar with this to reasoning the value.